packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/nodejs/githubinstructions.md

```diff
@@ -2,14 +2,14 @@ After you satisfy the software requirements, clone the
 [Okta Auth JavaScript SDK](https://github.com/okta/okta-auth-js)
 repo in a local directory on your computer.
 
-```bash
+```console
 git clone github https://github.com/okta/okta-auth-js.git
 cd okta-auth-js
```

Comment:
as this guide is targeting to the sample app, how about `cd` to the sample app directory?

```
cd okta-auth-js/samples/generated/express-embedded-sign-in-widget
```